examples/rust-examples/risc-v-esp32-c6/dynamic-memory/src/main.rs

```diff
@@ -8,8 +8,10 @@ use core::sync::atomic::{AtomicU32, Ordering};
 use esp_backtrace as _;
 use esp_hal::entry;
 use esp_println::println;
-use martos::init_system;
-use martos::task_manager::TaskManager;
+use martos::{
+    init_system,
+    task_manager::{TaskManager, TaskManagerTrait},
```

Comment:
For what `TaskManagerTrait` is required?

Reply:
It makes `start_task_manager` and `add_task` functions visible